lib/src/main/kotlin/io/bloco/faker/components/Business.kt

```diff
@@ -0,0 +1,24 @@
+package io.bloco.faker.components
+
+import io.bloco.faker.FakerComponent
+import io.bloco.faker.FakerData
+import java.time.LocalDate
+
+class Business(data: FakerData) : FakerComponent(data) {
+    fun creditCardNumber(): String {
+        return fetch("business.credit_card_numbers")
+    }
+
+    fun creditCardExpireDate(): LocalDate {
```

Comment:
@sdsantos just a question... this means it cant be use in KMM right? 
I know it would be a big change but...

Reply:
like this there are other still using java